packages/core/src/metrics/impl/DeviceMetrics.ts

```diff
@@ -0,0 +1,20 @@
+ import {Metrics} from '../Metrics';
+
+
+/**
+ * Collect device metrics:
+ *
+ *  - platform - ios
+ *  - platformVersion - version of the ios platform
+ *  - device - device name
+ */
+ export class DeviceMetrics implements Metrics {
+  public identifier: string = 'device';
+  public collect(): any {
+    return {
+      'platform': 'js',
+      'platformVersion': 1,
+      'device': 'TODO - cordova/react wrapper'
```

Comment:
@pb82 @david-martin  this cold be actually really good use case for react native/cordova wrapper support.
Pinging you for awareness.